index.md

```diff
@@ -238,3 +238,67 @@ Our typical usages of AWS include:
 1. S3 for hosting uploaded files
 1. ElastiCache instances for hosting in-memory data stores
 1. OpsWorks for provisioning each if the size or complexity is large
+
+# Documentation
+
+Traditionally documentation has been a second-class citizen in most Rails project. This holds true just as much outside The Nerdery as internally. As innovators and leaders in our industry we should hold ourselves to a higher standard and therefore provide ample documentation on the products we produce. A great article on improving Rails based documentation published by Aaron Sumner can be found [here](http://everydayrails.com/2016/04/18/rails-documentation-practices.html) and many of the practices described below are pulled from his philosophy.
```

Comment:
I feel this reads a little unprofessionally with the focus on *should*, like we don't live up to our own standards. What about something like this?

> Good documentation makes a long-lived project easier to understand and maintain. It can help introduce new developers to an unfamiliar codebase, or serve as a reference point for the team coming back to code 6 months later. Documentation may also cover important architectural decisions, complex business logic, or quirks and workarounds of the tech we're working with. Just as we put effort into making tests that last to ensure our application is stable and robust, we also focus on providing meaningful documentation.
>
> A great article on improving Rails-based documentation published by Aaron Sumner can be found [here](http://everydayrails.com/2016/04/18/rails-documentation-practices.html). Many of the practices described below are pulled from his philosophy.